packages/terra-consumer-nav/src/components/modal/Modal.scss

```diff
@@ -52,6 +53,9 @@
   .close-button {
     color: var(--terra-consumer--modal-close-button-color, #fff) !important;
     font-size: 16px !important;
+    margin: 0;
+    padding: 0;
```

Comment:
i seems @alex-bezek is working on the same fix for dex?

Reply:
@alex-bezek  we probably need !important here don't we?

Reply:
yes. anything being passed to a terra component via a class needs important